tests/test_install.ts

```diff
@@ -187,6 +188,34 @@ describe("Agama test", function () {
 
     // For development will be useful to stop before starting installation
     if (agamaInstall === true) {
+        it("should create first user", async function (){
+            await page.locator("a[href='#/users']").click();
+
+            let button: any = await Promise.any([
+                page.waitForSelector("a[href='#/users/first']"),
+                page.waitForSelector("button#actions-for-" + agamaUser)
+            ]);
+
+            await button!.click();
+
+            const id = await button!.evaluate((x: { id: any; }) => x.id);
+            // drop the handler to avoid memory leaks
+            button!.dispose();
+
+            // if the menu button was clicked we need to additionally press the "Discard" menu item
+            if (id === "actions-for-" + agamaUser) {
+                await page.locator("button[role='menuitem']::-p-text('Discard')").click();
+                await page.locator("a[href='#/users/first']").click();
+            }
+
+            await page.locator("input#userFullName").fill(agamaUserFullName);
+            await page.locator("input#userName").fill(agamaUser);
+            await page.locator("input#password").fill(agamaUserPassword);
+            await page.locator("input#passwordConfirmation").fill(agamaUserPassword);
+            await page.locator("button[form='firstUserForm']").click();
+            await page.locator("a[href='#/overview']").click();
```

Comment:
blank line above this one to separate logical actions in block of code.

Reply:
Solved bbf69efe39e06cc95d295f663d3b220933fc32d0